src/app/public/modules/confirm/confirm-config.ts

```diff
@@ -7,9 +7,28 @@ import {
 } from './confirm-button-config';
 
 export interface SkyConfirmConfig {
+  /**
+   * Specifies the message to display in a bold font at the top of the dialog.
+   * @required
```

Comment:
We have the `@required` parameter, but this property is not displayed before the optional parameters and there is no "Required" label. @Blackbaud-SteveBrush, are we doing something wrong here? Or should we file an issue in the `docs-tools` repo?